React/Views/RCTPickerManager.m

```diff
@@ -19,7 +19,9 @@ @implementation RCTPickerManager
 
 - (UIView *)view
 {
-  return [RCTPicker new];
+  RCTPicker *picker = [RCTPicker new];
+  [picker selectRow:0 inComponent:0 animated:YES];
```

Comment:
We definitely should not configure it here.

Reply:
Hi, 

Is there any update on this please I am trying to use picker and the lines disappear when I interact with it...
Thanks

Reply:
The fix works fine within React/Views/RCTPickerManager.m but when I move it within  RCTPicker's initWithFrame it doesn't work. 

<img width="327" alt="screen shot 2018-03-01 at 13 39 10" src="https://user-images.githubusercontent.com/27201465/36847492-705ff3d4-1d56-11e8-9afe-5bcaf86b1860.png">